src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/config/QueueClientsConfiguration.java

```diff
@@ -10,7 +10,7 @@
 import org.springframework.context.annotation.Profile;
 
 @Configuration
-@Profile("!nosb") // do not register handler for the nosb (test) profile
+@Profile("!functional & !integration") // do not register handler for the 'functional' (test) profile
```

Comment:
same as above

Reply:
If this stays like that, the comment needs to be updated or removed - these profiles are not strictly related with any handlers